src/dispatcher/dispatch-actions/threads.ts

```diff
@@ -18,22 +18,31 @@ import { MatrixEvent } from "matrix-js-sdk/src/models/event";
 import RightPanelStore from "../../stores/right-panel/RightPanelStore";
 import { RightPanelPhases } from "../../stores/right-panel/RightPanelStorePhases";
 
-export const dispatchShowThreadEvent = (
+export const showThread = (
     rootEvent: MatrixEvent,
     initialEvent?: MatrixEvent,
     highlighted?: boolean,
+    push = false,
```

Comment:
Could we change this function to accept an object with these as properties? That way it will be clear what each value is at the call site, and unused optional values can just be omitted.

Reply:
I was going back and forth with this. Its good to have a second opinion. When using it more I also had the feeling that having named params would make everything clearer.